simulators/active_surface.py

```diff
@@ -18,29 +18,29 @@ class Driver(object):
     }
 
     def __init__(self, driver_reset_delay=0):
-        self.driver_reset_delay = driver_reset_delay  #Actually it should be 100ms
+        self.driver_reset_delay = driver_reset_delay  # Actually it should be 100ms
         self.reset()
 
     def reset(self):
         self.reference_position = 0
         self.current_position = 0
         self.position_queue = Queue.Queue()
 
-        self.delay_multiplier = 0   # x*delay_step. 255=infinite delay (no response)
+        self.delay_multiplier = 0  # x*delay_step. 255=infinite delay (no response)
 
-        self.version = [1, 3]       # Major, minor
-        self.driver_type = 0x20     # 0x20 for USD50xxx, 0x21 for USD60xxx
+        self.version = [1, 3]  # Major, minor
+        self.driver_type = 0x20  # 0x20 for USD50xxx, 0x21 for USD60xxx
 
         self.slope_multiplier = 1
 
         self.min_frequency = 20
         self.max_frequency = 10000
 
-        self.IO_dir = [0, 0, 0]     # 0 = input, 1 = output
-        self.IO_val = [0, 0, 0]     # These values show the corresponding
-                                    # I/O line value, but only if the direction
-                                    # of the I/O line is set to output
-                                    # (IO_dir[x] = 1)
+        self.IO_dir = [0, 0, 0]  # 0 = input, 1 = output
+        self.IO_val = [0, 0, 0]  # These values show the corresponding
```

Comment:
This comment should be in the line right before the `self.IO_val` assignment, written in this way:
```
# These values show the corresponding I/O line value, but only if the direction
# of the I/O line is set to output (IO_dir[x] = 1)
self.IO_val = [0, 0, 0]
```